flow/.eslintrc.json

```diff
@@ -15,7 +16,9 @@
   "rules": {
     "flowtype/no-types-missing-file-annotation": "off",
     "flowtype/generic-spacing": "off",
-    "react/no-unsafe": [2, { "checkAliases": true }]
+    "react/no-unsafe": [2, { "checkAliases": true }],
+    "jsx-a11y/no-autofocus": "off",
+    "jsx-a11y/label-has-associated-control": "off"
```

Comment:
Why are we setting this lint off? 🤔

Reply:
This is an accessible option (https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/master/docs/rules/label-has-associated-control.md):
```
<label>
  Surname
  <input type="text" />
</label>
```

And this is how we are doing.
But as our inputs are normally components, the lint is not checking this correctly, so we have disabled it for now.

Reply:
Got it.
What about thinking in a strategy to remove this lint disable later? =)
I would happily help with this.